packages/form-participation/test-web/utils.test.ts

```diff
@@ -0,0 +1,119 @@
+import { expect, fixture, fixtureCleanup, html } from '@open-wc/testing';
+import { spy } from 'sinon';
+import { formValues, parseFormAsObject, submit } from '../src/utils';
+
+describe('The implicit submit function', () => {
+  let form: HTMLFormElement;
+
+  beforeEach(async () => {
+    form = await fixture(html`<form>
+      <input name="foo">
+    </form>`) as unknown as HTMLFormElement;
+
+    form.submit = function() {
+      form.submitted = true;
+    }
+  });
+
+  afterEach(fixtureCleanup);
+
+  it('calls reportValidity', async () => {
+    const reportValiditySpy = spy(form, 'reportValidity');
+    submit(form);
+    expect(reportValiditySpy.called).to.be.true;
+    expect(form.submitted).to.be.true;
+  });
+
+  it('emits a submit event', async () => {
+    const dispatchEventSpy = spy(form, 'dispatchEvent');
+    const submitEvent = new Event('submit', {
+      cancelable: true
+    });
+    submit(form);
+    expect(dispatchEventSpy.calledWith(submitEvent));
+    expect(form.submitted).to.be.true;
+  });
+
+  it('will not proceed if the form is invalid', async () => {
+    const reportValiditySpy = spy(form, 'reportValidity');
+    const dispatchEventSpy = spy(form, 'dispatchEvent');
+    form.querySelector('input').required = true;
+    submit(form);
+    expect(reportValiditySpy.called).to.be.true;
+    expect(dispatchEventSpy.called).to.be.false;
+  });
+
+  it('will not submit if event is prevented', async () => {
+    const prevent = (event: Event) => event.preventDefault();
+    form.addEventListener('submit', prevent);
+    submit(form);
+    expect(form.submitted).to.equal(undefined);
+    form.removeEventListener('submit', prevent);
+  });
+});
+
+
+describe('The formValues utility', () => {
+  let form: HTMLFormElement;
+
+  beforeEach(async () => {
+    form = await fixture(html`<form>
```

Comment:
```suggestion
    form = await fixture<HTMLFormElement>(html`<form>
```
Does this prevent the need to cast through `unknown`?

Reply:
Yes. Updated.